### DISCUSSION TITLE
[Porting OG hyprdots]: sort prs currently in hyprdots

### DISCUSSION THREAD

**Discussion by abenezerw** [HyDE-Project/HyDE#20] [Polls]
Make sure to have a list guys. 
1 for vital 
0 for fluff/unnecessary 
2 for up for voting it in

**Comment by abenezerw** [comment]
1 for external monitor brightness support #1771 , for people with external monitors. 
1 for pokeget instead of colorscripts #1894 I feel like kitty would load faster

**Reply by kRHYME7** [reply]
>  1 for pokeget instead of colorscripts #1894 I feel like kitty would load faster
Already done this
https://github.com/HyDE-Project/HyDE/blob/06041f629f603790649c9f160ef082ff6164cac6/Configs/.zshrc#L88C1-L89C27

> 1 for external monitor brightness support #1771 , for people with external monitors. 
Please open a separate issue for this one

**Reply by kRHYME7** [reply]
@abenezerw  I added `ddcui ` for now. It's the most responsive one.  for integration on the script we should find other method. ddcutil is so slow.

**Comment by rubiin** [comment]
- [x] https://github.com/prasanthrangan/hyprdots/pull/2008
- [x] https://github.com/prasanthrangan/hyprdots/pull/2009
- [ ] https://github.com/prasanthrangan/hyprdots/pull/2049
- [ ] https://github.com/prasanthrangan/hyprdots/pull/1818

**Reply by kRHYME7** [reply]
> https://github.com/prasanthrangan/hyprdots/pull/2008

done

> https://github.com/prasanthrangan/hyprdots/pull/2009

~should be added to custom_app.lst. 
FYI nwg-look autogenerates the configs so I won't consider it as a core package. 
Added it in app.lst~
Ops yeah should be in core

> https://github.com/prasanthrangan/hyprdots/pull/2049

Feat Req please, I already added the hyprsunset.sh but I hate killing hyprsunset just to change values. 

> https://github.com/prasanthrangan/hyprdots/pull/1818

Done. 
Added a reference file at https://github.com/HyDE-Project/HyDE/blob/master/Configs/.config/hypr/hyprland.conf

**Comment by kRHYME7** [comment]
Let's utilize the discussion panel.
